stack/values.yaml

```
@@ -39,6 +39,14 @@ promscale:
     requests:
       memory: 500Mi
       cpu: 30m
+  config:
+    startup.dataset.config: |
+      metrics:
+        default_chunk_interval: 1h
+        compress_data: true
+        default_retention_period: 90d
+      traces:
+        default_retention_period: 30d
```

Can we limit this only to overrides without specifying default values?

Since this is introducing tech-debt on promsclae side (we are testing with non-default values), could you create an issue in the promscale project so we can keep track of this?

---

Done. https://github.com/timescale/promscale/issues/1736

---

```suggestion
      metrics:
        default_chunk_interval: 1h
```

Let's set only values which are not the default ones.

---

Created a new PR for this: https://github.com/timescale/promscale-benchmark/pull/31